test/src/unit-capi-config.cc

```diff
@@ -240,6 +240,9 @@ void check_save_to_file() {
   ss << "sm.consolidation.timestamp_start 0\n";
   ss << "sm.dedup_coords false\n";
   ss << "sm.enable_signal_handlers true\n";
+  ss << "sm.encryption_key 0\n";
+  ss << "sm.encryption_key_length 0\n";
```

Comment:
We should be able to infer the length from the value of `sm.encryption_key`. Please remove this config value.